src/Symfony/Component/Form/Form.php

```diff
@@ -831,7 +831,7 @@ public function all()
     /**
      * {@inheritdoc}
      */
-    public function add($child, $type = null, array $options = [])
+    public function add($child, string $type = null, array $options = [])
```

Comment:
$type is also referencing `FormTypeInterface` in code which can be removed and should be fixed in 4.4

Reply:
#32934